pull-request.py

```diff
@@ -395,7 +395,10 @@ def main():
         print("PULL_REQUEST_FROM_BRANCH is not set, checking branch in payload.")
         with open(check_events_json(), "r") as fd:
             from_branch = json.loads(fd.read()).get("ref")
```

Comment:
Thank you for catching this! A question about your choice of fix - why not just do:

```suggestion
            from_branch = json.loads(fd.read()).get("ref", "")
```
And avoid the extra try / except and exception? The result of the above is an empty string, which won't trigger the error, and will cleanly exit in the next block telling the user to set the envar.

Reply:
Good point, that is cleaner. Updating now